components/checkbox/index.css

```
@@ -271,51 +433,37 @@
 		}
 	}
 
-	/* Hover for Checked and Indeterminate States */
-	&:hover {
-		.spectrum-Checkbox-input:checked + .spectrum-Checkbox-box,
-		&.is-indeterminate .spectrum-Checkbox-input:checked + .spectrum-Checkbox-box,
-		&.is-indeterminate .spectrum-Checkbox-box {
-			&::before {
-				border-color: var(--highcontrast-checkbox-color-hover, var(--mod-checkbox-emphasized-color-hover, var(--spectrum-checkbox-emphasized-color-hover)));
-			}
-		}
-	}
-
-	/* Selected and Indeterminate Focus States */
-	.spectrum-Checkbox-input:focus-visible:checked + .spectrum-Checkbox-box,
-	&.is-indeterminate .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-box {
+	&:not(.is-disabled).spectrum-Checkbox-input:checked:focus-visible + .spectrum-Checkbox-box {
 		&::before {
-			border-color: var(--highcontrast-checkbox-highlight-color-focus, var(--mod-checkbox-emphasized-color-focus, var(--spectrum-checkbox-emphasized-color-focus)));
+			border-color: var(--highcontrast-checkbox-highlight-color-default, var(--mod-checkbox-emphasized-color-down, var(--spectrum-checkbox-emphasized-color-down)));
 		}
 	}
 
-	/* Selected Invalid Focus State */
-	&.is-invalid .spectrum-Checkbox-input:focus-visible:checked + .spectrum-Checkbox-box {
+	&.is-indeterminate .spectrum-Checkbox-input.is-focus-visible + .spectrum-Checkbox-box {
```

```suggestion
	&.is-indeterminate .spectrum-Checkbox-input:focus-visible + .spectrum-Checkbox-box {
```
This one references the Storybook-only testing class as well.